packages/framework-core/src/services/event-store.ts

```diff
@@ -206,4 +227,24 @@ export class EventStore {
       }
     }
   }
+
+  private eventMetadata(eventEnvelope: EventEnvelope): EventMetadata | UnknownEventMetadata | undefined {
+    const event = this.config.events[eventEnvelope.typeName]
+    if (event) return event
+    return this.config.unknownEvent
+  }
+
+  private isUnknownEventMetadata(
```

Comment:
I think we have the same method declared before in `booster-event-dispatcher.ts`. If is the same method maybe we should move it to a common place.